Upgrade to Spark-3.3.2.

kgabor opened the pull request:
* Manual merge of pom changes by @StephanPreibisch.
 * Upgrade pom to released n5 packages and removing unnecessary versions in the new scijava.
 * Fix duplicate classes in pom.
 * Remove unnecessary explicit versions, exclusions.
 * Additional pom cleanup.

 @StephanPreibisch I upgraded the dependencies to Spark-3.3.2 on top of your `mvr` and related upgrade. Tested with simple, fatjar build and direct `java` and `spark-submit` runs. 

@trautmane The fatjar creation complained about the duplication of the following meta files. As far as I understand the first two items come from the different java version kotlin modules which may not be the used at the same time. I'm unsure about the `jackson` one. Is it OK to simply exclude these from the fatjar ?

```xml
<exclude>META-INF/versions/9/module-info.class</exclude>
<exclude>META-INF/okio.kotlin_module</exclude>
<exclude>META-INF/services/com.fasterxml.jackson.core.JsonFactory</exclude>
```

tpietzsch commented on pom.xml:
```diff
@@ -121,96 +118,50 @@
             <groupId>org.bigdataviewer</groupId>
             <artifactId>bigdataviewer-omezarr</artifactId>
-            <version>0.1.1</version>
+            <version>0.1.2-keyvalue-SNAPSHOT</version>
```
@kgabor You should make a proper release and depend on that

kgabor commented:
* Released ``bigdataviewer-omezarr-0.2.0`` that depends on the ``keyvalue`` n5 suite (``n5-3.0.0``).
* Updated the pom in this PR accordingly. 
* Updated the scijava base and removed unnecessary explicit versions where it matches the new default.

kgabor commented:
@tpietzsch I'll check the pom details with @trautmane . Then I think this should be ready for merging and it should be done before things start diverging again.
